zenoh_router/README.md

```diff
@@ -0,0 +1,84 @@
+# Agent Check: Zenoh router
+
+## Overview
+
+This check monitors Zenoh router.
+
+[Zenoh][1] is open source Zero Overhead Network Protocol.
+
+Zenoh /zeno/ is a pub/sub/query protocol unifying data in motion, data at rest and computations. It elegantly blends traditional pub/sub with geo distributed storage, queries and computations, while retaining a level of time and space efficiency that is well beyond any of the mainstream stacks.
+
+Zenoh router integration allows get statistic from zenoh router and monitor router/peer/client connection statuses.
+
+## Setup
+
+### Installation from the datadog
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install Zenoh router check on your host.
```

Comment:
I recommend omitting this line, since the heading and the edits below now cover it.